src/graphnet/data/extractors/i3splinempeextractor.py

```diff
@@ -0,0 +1,22 @@
+from graphnet.data.extractors.i3extractor import I3Extractor
+
+class I3SplineMPEICExtractor(I3Extractor):
+    def __init__(self, name="spline_mpe_ic"):
+        super().__init__(name)
+
+    def __call__(self, frame) -> dict:
+        """Extracts TUM DNN Recos and associated variables"""
+        output = {}
+        if "SplineMPEIC" in frame:
+            output.update(
+                {
+                    "zenith_spline_mpe_ic": frame[
+                        "SplineMPEIC"
+                    ].dir.zenith,
+                    "azimuth_spline_mpe_ic": frame[
+                        "SplineMPEIC"
+                    ].dir.azimuth,
+                }
+            )
```

Comment:
Man, sometimes `black` gets on my nerves...

Reply:
I feel you